src/runtime/offers-flow-test.js

```diff
@@ -156,6 +156,32 @@ describes.realWin('OffersFlow', {}, env => {
     return offersFlow.start();
   });
 
+  it('should have valid OffersFlow constructed with skus and oldSku', () => {
```

Comment:
add another test with expected behavior when a list of skus is not present

Reply:
Added, will be in next PR!

Reply:
And pushed!